### DISCUSSION TITLE
New features: show_backlinks() and find_friends()

### DISCUSSION THREAD

**Discussion by renerocksai** [nvim-telekasten/telekasten.nvim#8] [Announcements]
See #3 from @lokesh-krishna: this function pops up a telescope search finding all notes that link to the current one - aka 'BACKLINKS'.

Run it like this:

```vim
:lua require("telekasten").show_backlinks()
```

Or bind it like this:

```vim
nnoremap <leader>zb :lua require('telekasten').show_backlinks<CR>
```

Thanks for the great suggestion! :+1:

**Comment by renerocksai** [comment]
`find_friends()` pops up a telescope search finding all notes that link to the same note as the link under the cursor.

So if you have a block like this:

```markdown
See also ........... [[some other note]]
```

Then placing the cursor on the `[[some other note]]` link and running `lua require('telekasten').find_friends()` will search for all notes that also link to `[[some other note]]`.

**Comment by lokesh-krishna** [comment]
Oooh, that's awesome!
